src/AutoROM.py

```diff
@@ -197,6 +197,23 @@ def torrent_tar_to_buffer():
 
     return buffer
 
+def verify_installation(package, checksum_keys):
+    for file in os.listdir(package):
+        if not file.endswith(".bin"):
+            continue
+
+        rom_path = os.path.join(package, file)
+        hash = hashlib.md5(open(rom_path,'rb').read()).hexdigest()
+
+        if not hash in checksum_keys:
+            return False
+
+        checksum_keys.remove(hash)
```

Comment:
As `checksum_keys` is a reference then we are modifying this reference, are we sure this doesn't mess something else up?

Reply:
[Keys returns a copy](https://python-reference.readthedocs.io/en/latest/docs/dict/keys.html)